### PR TITLE
Edited kovri script

### PR DESCRIPTION
**Pull request by ajs-xmr**

### CODE REVIEW

File: kovri.md [hunk 1]
```diff
@@ -1,55 +1,51 @@
...
-Some of the nodes on the network may be logging the IP addresses they get when transaction requests are given.
-An IP address is like a home address for the internet.
-Attackers will try to associate transactions with the IP addresses they originate from.
-They do not know anything about what address a transaction is coming from, where it is going to, or what the amount is.
+Up to now, we have covered how Monero obfuscates information stored on the blockchain.
+Ring signatures obscure the sender.
+Stealth addresses prevent outside observers from knowing the destination of where Monero is sent to.
+And RingCT hides the amount of Monero that's transmitted.
+However, some personally identifiable information may be leaked at the network level when making a transaction.
+This is where Kovri comes in to fill this possible gap.
```
Comment (anonimal) [inline kovri.md]: >This is where Kovri comes in to fill this possible gap.

I'd prefer to see another wording/phrasing of this (either before or after merge).

File: kovri.md [hunk 2]
```diff
@@ -1,55 +1,51 @@
...
 
-However, an attacker with enough resources could knock on Alice's door and demand that she tell them this information.
-Alternatively, they could purposefully attempt to block her transaction from being broadcast to the rest of the network.
+## 3) - The Term - Kovri
 
-So what can Alice do to mitigate these threats? Well, she can use Kovri!
+Kovri is a free, decentralized application, which is based on the Invisible Internet Project's I2P open specifications.
```
Comment (anonimal) [inline kovri.md]: >Invisible Internet Project's I2P

You're repeating yourself: I2P is an acronym for Invisible Internet Project.

File: kovri.md [hunk 3]
```diff
@@ -1,55 +1,51 @@
...
+Kovri uses both encryption and innovate routing techniques to create a private overlay-network across the Internet.
```
Comment (anonimal) [inline kovri.md]: >innovate

Though worth praise, I wouldn't consider 15+ year-old routing technology (onion routing, the basis of garlic routing) to be very innovative in current year. Also, misspell of 'innovative'.

File: kovri.md [hunk 4]
```diff
@@ -1,55 +1,51 @@
...
-Alice can optionally send all the Monero sync data, not just the transaction broadcast, through Kovri.
-Now, her Monero connections are hidden from her ISP.
-This improves Monero's censorship resistance, since other people no longer know who is running a Monero node, and others cannot demand that these nodes be shut down.
-
-Let’s look at one last scenario.
-Suppose Charlie is the operator of a mining pool that controversially donates a portion of the block reward to a political party.
-Other nodes could purposefully reject his solved blocks to express their disagreement with this policy.
-This sort of attack is called a Sybil attack.
-However, if Charlie can send his solved blocks through Kovri, then no one can censor the work from his pool, since they no longer know that his pool created it.
+Suppose Charlie has an Internet Service Provider that does not want him to use Monero and he receives a warning to cease and desist in the mail.
```
Comment (anonimal) [inline kovri.md]: >Suppose Charlie is the operator of a mining pool
>Suppose Charlie has an Internet Service Provider

Charlie was specifically an operator of a mining pool. In your revision, Charlie has no context; he could be anyone. This is not the definition of a Charlie as 3rd person in relation to Alice and Bob (as was the operator of a mining pool).

This sentence is also seems more relevant for Alice who runs her own node and uses her wallet locally, yes? And/or Bob who syncs nodes with other Bobs?

File: kovri.md [hunk 5]
```diff
@@ -1,55 +1,51 @@
...
+Charlie could optionally mask his Internet traffic through Kovri and in doing so, prevent others from knowing he is running a Monero node.
+This substantially improves Monero's censorship resistance by making it even more difficult to detect and shut down nodes.
 
-Kovri will come bundled with future Monero releases, and it will be enabled by default to broadcast the transaction anonymously.
-Kovri will feature a common API that can be used by any cryptocurrency, not just Monero.
+In our last scenario, suppose David is the operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
```
Comment (anonimal) [inline kovri.md]: See Charlie comment above.

**Review by anonimal — CHANGES_REQUESTED** [review]

File: kovri.md [hunk 6]
```diff
@@ -9,12 +9,12 @@ Ring signatures obscure the sender.
 Stealth addresses prevent outside observers from knowing the destination of where Monero is sent to.
 And RingCT hides the amount of Monero that's transmitted.
 However, some personally identifiable information may be leaked at the network level when making a transaction.
-This is where Kovri comes in to fill this possible gap.
+This gap of privacy is closed by a technology called Kovri.
```
Comment (SamsungGalaxyPlayer) [inline kovri.md]: This sentence sounds so weird to me.

File: kovri.md [hunk 7]
```diff
@@ -32,13 +32,14 @@ So what can Alice do to mitigate these threats? Well, she can use Kovri!
...
+After a few weeks, she receives a cease and desist letter from her Internet Service Provider informing her that a running node is a violation of the terms of service.
+Alex could have avoided this situation by masking her Internet traffic with Kovri.
 This substantially improves Monero's censorship resistance by making it even more difficult to detect and shut down nodes.
 
-In our last scenario, suppose David is the operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
+In our last scenario, suppose Andrew is an operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
 Other nodes could purposefully reject his solved blocks to express their disagreement with his political views.
-However, if David can send his solved blocks through Kovri, then no one can censor the work from his pool, since the source of solved blocks becomes unknown.
+However, if Andrew can send his solved blocks through Kovri, then no one can censor the proof of work from his pool, since the source of solved blocks are unknown.
```
Comment (SamsungGalaxyPlayer) [inline kovri.md]: Thoughts on changing this to "no one can censor the new blocks created by his pool"?

File: kovri.md [hunk 8]
```diff
@@ -9,12 +9,12 @@ Ring signatures obscure the sender.
...
 And RingCT hides the amount of Monero that's transmitted.
 However, some personally identifiable information may be leaked at the network level when making a transaction.
-This is where Kovri comes in to fill this possible gap.
+This gap of privacy is closed by a technology called Kovri.
 
 ## 3) - The Term - Kovri
 
-Kovri is a free, decentralized application, which is based on the Invisible Internet Project's I2P open specifications.
-Kovri uses both encryption and innovate routing techniques to create a private overlay-network across the Internet.
+Kovri is a free, decentralized application, which is based on I2P.
```
Comment (anonimal) [inline kovri.md]: >which is based on I2P
>I2P open specifications

This needs to be very clear. It is not based on I2P, it is based on open I2P specifications. Your revision's language is less technical which is good for this document but we also need to be factually accurate.

We should keep the current official line of "Kovri is a free, decentralized, anonymity technology based on [I2P](https://getmonero.org/resources/moneropedia/i2p.html)'s open specifications".

File: kovri.md [hunk 9]
```diff
@@ -9,12 +9,12 @@ Ring signatures obscure the sender.
...
+Kovri uses both encryption and special routing techniques to create a private overlay-network across the Internet.
```
Comment (anonimal) [inline kovri.md]: s/special/complex/. Or another word that is accurate but sounds intriguing?

File: kovri.md [hunk 10]
```diff
@@ -32,13 +32,14 @@ So what can Alice do to mitigate these threats? Well, she can use Kovri!
 If Alice sends her transaction with Kovri, nodes will no longer see her IP address, making passive surveillance tactics impractical.
 
 Now let’s imagine a different scenario.
-Suppose Charlie has an Internet Service Provider that does not want him to use Monero and he receives a warning to cease and desist in the mail.
-Charlie could optionally mask his Internet traffic through Kovri and in doing so, prevent others from knowing he is running a Monero node.
+Suppose Alex wants to support the Monero network by running a full node at her home.
```
Comment (anonimal) [inline kovri.md]: Please don't change the name. See https://github.com/alvinjoelsantos/promo-video/pull/25#discussion_r135118023.

File: kovri.md [hunk 11]
```diff
@@ -32,13 +32,14 @@ So what can Alice do to mitigate these threats? Well, she can use Kovri!
...
+After a few weeks, she receives a cease and desist letter from her Internet Service Provider informing her that a running node is a violation of the terms of service.
+Alex could have avoided this situation by masking her Internet traffic with Kovri.
 This substantially improves Monero's censorship resistance by making it even more difficult to detect and shut down nodes.
 
-In our last scenario, suppose David is the operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
+In our last scenario, suppose Andrew is an operator of a mining pool that donates a portion of block rewards to a political party or controversial cause.
```
Comment (anonimal) [inline kovri.md]: Please don't change the name. See https://github.com/alvinjoelsantos/promo-video/pull/25#discussion_r135118023.

**Review by anonimal — CHANGES_REQUESTED** [review]

File: kovri.md (hunk 6 above)
Comment (anonimal) [reply kovri.md]: How about something like "This privacy leak is fixed by a technology called Kovri".

**Review by anonimal — APPROVED** [review]
@SamsungGalaxyPlayer voiced concern over not enough tech info. I'll take this to `#promo-video`.

**Pull request merged**